db/migrate/20171212192237_change_country_null_on_users.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeCountryNullOnUsers < ActiveRecord::Migration[5.1]
```

Comment:
Why not remove the column instead?

Reply:
The production DB has data in that column, I didn't know whether that would be an issue. I can remove it instead

Reply:
Is the data relevant? In other words, is there anything other than `Brasil` on that column?

Reply:
No, just that.

Reply:
Then let's remote the column ;)